finalPrep/modZeroHero.js

```diff
@@ -1,52 +1,93 @@
-//   fullEnergy should add 150 to your current energy
-
-// Declare two variables - isHuman and identityConcealed - assigned to booleans
-
-
-// Declare two variables - archEnemies AND sidekicks
-//   archEnemies should be an array of at least 3 different enemy strings
-//   sidekicks should be an array of at least 3 different sidekick strings
-
-// Print the first sidekick to your console
-
-// Print the last archEnemy to the console
+// Declare two variables - heroName AND specialAbility - set to strings √
+var heroName = "Batman";
+var specialAbility = "Cunning";
+
+// Declare two variables - greeting AND catchphrase √
+//   greeting should be assigned to a string that uses concatenation to include the heroName √
+//   catchphrase should be assigned to a string that uses interpolation to include the specialAbility √
+var greeting = "Hello "+ heroName;
+var catchphrase =`${specialAbility} is my weapon`;
+
+// Declare two variables - power AND energy - set to integers √
+var power = 77;
+var energy = 89;
+
+// Declare two variables - fullPower AND fullEnergy √
+//   fullPower should multiply your current power by 500 √
+//   fullEnergy should add 150 to your current energy √
+var fullPower = 500 * power;
+var fullEnergy = 150 + energy;
+
+// Declare two variables - isHuman and identityConcealed - assigned to booleans √
+var isHuman = true;
+var identityConcealed = false;
+
+// Declare two variables - archEnemies AND sidekicks √
+//   archEnemies should be an array of at least 3 different enemy strings √
+//   sidekicks should be an array of at least 3 different sidekick strings √
+var archEnemy = ["Jocker", "Two-Face", "Riddler"];
+var sidekicks =["Robin", "Catwoman", "Alfred"] ;
+
+// Print the first sidekick to your console √
+console.log(sidekicks[0]);
+
+// Print the last archEnemy to the console √
+console.log(archEnemy[2]);
 
 // Write some code to add a new archEnemy to the archEnemies array
+archEnemy.push("Penguin");
 
 // Print the archEnemies array to console to ensure you added a new archEnemy
-
+console.log(archEnemy);
 // Remove the first sidekick from the sidekicks array
-
+sidekicks.shift();
 // Print the sidekicks array to console to ensure you added a new sidekick
+console.log(sidekicks);
 
-// Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
-//   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
-//   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
+// Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse √
+//   - dangerLevel should be an integer √
+//   - saveTheDay should be a string a hero would say once they save the day √
+//   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel √
+function assessSituation(dangerLevel, saveTheDay, badExcuse){
```

Comment:
I expected to see the if/else statement _written within the function_ you defined (the `assessSituation` function).  For example:
```
function assessSituation(dangerLevel, saveTheDay, badExcuse) {
  if (<YOUR EXPRESSION HERE>) {
    console.log(badExcuse);
  } else if (<YOUR EXPRESSION HERE>){
    console.log(saveTheDay);
  } else {
    console.log("Meh. Hard pass.")
  }
}
```